libs/shared/contentful/src/lib/contentful-client.spec.ts

```diff
@@ -38,6 +38,12 @@ describe('ContentfulClient', () => {
     Object.defineProperty(contentfulClientConfig, 'graphqlApiUri', {
       value: faker.string.uuid(),
     });
+    Object.defineProperty(contentfulClientConfig, 'graphqlSpaceId', {
+      value: faker.string.uuid(),
+    });
+    Object.defineProperty(contentfulClientConfig, 'graphqlEnvironment', {
+      value: faker.string.uuid(),
+    });
```

Comment:
I was talking with @julianpoy after my `ContentfulClientConfig` [polish PR](https://github.com/mozilla/fxa/pull/15727) went through, and this can actually be simplified to something like:

```ts
contentfulClientConfig = new ContentfulClientConfig({
  graphqlApiKey: faker.string.uuid(),
  graphqlApiUri: faker.string.uuid(),
  // other ones...
});
```

Could you make that small improvement while you're in here?